react/index.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import FormInput from 'vtex.instore-form-fields/FormInput'
 
-class StoneFields extends React.Component {
-  render() {
-    return null
-  }
+const StoneFields = () => {
+  return (
+    <Fragment>
+      <FormInput
+        ffor="customsenha"
```

Comment:
where do these values in `ffor` come from?

Reply:
I don't know, really. I just copy and pasted the values that were in the `pages.json` before.